sdk/src/types/block/slot/commitment.rs

```diff
@@ -18,6 +18,9 @@ use crate::{
     serde(rename_all = "camelCase")
 )]
 pub struct SlotCommitment {
+    // The version of the protocol running.
+    #[serde(rename = "version")]
+    protocol_version: u8,
```

Comment:
I renamed it to version in the params because it's the protocol params, so the protocol part is implied. Here, I'm not sure that logic applies. If I had my way I would change it in the TIP.

Reply:
Fair enough, maybe suggest it in the TIP?

Reply:
Done